### PR TITLE
[DF-02] Prompt update

### PR DESCRIPTION
**Pull request by PyyLog**
YouTrack: [DF-02](https://irisrobotics.youtrack.cloud/issue/PMI-16/Dataset-input-output)

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -30,23 +33,23 @@ def model(self):
                     "content": self.prompt,
                 },
             ],
-            temperature=0.3,
+            temperature=0.5,
```
Comment (PhantHive) [inline main.py]: You got better result with this temperature? 
Did you try other value?

File: prompt.py [hunk 2]
```diff
@@ -1,27 +1,59 @@
...
+    You MUST NOT create requests that are too general.\n
+    output should be an answer to the input as a JSON format.\n"
+    \nGive me a JSON format that is an array of EXACTLY %s lines (instructions,inputs,outputs) this way:"
+    [
+             {
+                instruction: "You are a data parser aim to help students finding resources, answer with a json format
+                as best as possible.",
+                input: "I need help with mathematics",
+                output: {
+                    "topic": ...,
```
Comment (PhantHive) [inline prompt.py]: Why not indicating a real example (topic, subject)

File: main.py [hunk 3]
```diff
@@ -2,11 +2,12 @@
 import os
 import openai
 from dotenv import load_dotenv
-from prompt import prompt
+import prompt
```
Comment (PhantHive) [inline main.py:5]: AHAA nice, I don't recall why I did this stupid "from A import A" 😆

File: main.py (hunk 1 above)
Comment (PyyLog) [reply main.py]: I tried .5 to see if the generated inputs would be more creative.... For example, for the electromagnetism subject, the input created by GPT was "Pourriez-vous me donner des conseils sur le sujet de l'Ampère ?" which is not relevant at all.

The result with this change is that the inputs are still the same in term of specificity (too general).
Just forgot to put it back to .3 :)

File: prompt.py (hunk 2 above)
Comment (PyyLog) [reply prompt.py]: You're right ! I put three dots to not let GPT build inputs about that topic in the example. I did this before adding some details in the prompt to be more specific about the subject to tackle and its related topics.

**Comment by PyyLog** [comment]
I will translate the prompt in french to get an instruction and input in french and keep the output in english.
At the same time, i will remove the loop for iterations as in the prompt, I already specified a number of inputs to be created.

**Review by KreatorOf — COMMENTED** [review]
all checked!

**Review by KreatorOf — APPROVED** [review]

File: DataForgeConfig.json [hunk 4]
```diff
@@ -0,0 +1,8 @@
+{
+  "nb_observations": 5,
+
+  "themes_dict": {
+    "Maths": ["Matrices", "systemResolution"],
+    "Electromag": ["Ampere", "loadDistribution"]
```
Comment (KreatorOf) [inline DataForgeConfig.json:6]: Might be useful to change the themes.

**Pull request merged**